pygmt/src/subplot.py

```diff
@@ -0,0 +1,215 @@
+            lib.call_module(module="subplot", args=arg_str)
+            yield
+        finally:
+            v_arg = build_arg_string({"V": kwargs.get("V")})
+            lib.call_module("subplot", f"end {v_arg}".strip())
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(A="fixedlabel", C="clearance", V="verbose")
+def set_panel(self, panel=None, **kwargs):
+    r"""
+    Set the current subplot panel to plot on.
+
+    Before you start plotting you must first select the active subplot. Note:
+    If any *projection* option is passed with **?** as scale or width when
+    plotting subplots, then the dimensions of the map are automatically
+    determined by the subplot size and your region. For Cartesian plots: If you
+    want the scale to apply equally to both dimensions then you must specify
+    ``projection="x"`` [The default ``projection="X"`` will fill the subplot by
+    using unequal scales].
+
+    {aliases}
+
+    Parameters
+    ----------
+    panel : str or list
+        *row,col*\|\ *index*.
+        Sets the current subplot until further notice. **Note**: First *row*
+        or *col* is 0, not 1. If not given we go to the next subplot by order
+        specified via **autolabel** in :meth:`pygmt.Figure.subplot`. As an
+        alternative, you may bypass the **sca** mode and instead supply the
+        common option **panel**=\ [*row,col*] to the first plot command you
+        issue in that subplot. GMT maintains information about the current
+        figure and subplot. Also, you may give the one-dimensional *index*
+        instead which starts at 0 and follows the row or column order set via
+        **autolabel** in :meth:`pygmt.Figure.subplot`.
+
+    fixedlabel : str
+        Overrides the automatic labeling with the given string. No modifiers
+        are allowed. Placement, justification, etc. are all inherited from how
+        **autolabel** was specified by the initial :meth:`pygmt.Figure.subplot`
+        command.
+
+    clearance : str
+        [*side*]\ *clearance*.
+        Reserve a space of dimension *clearance* between the margin and the
+        subplot on the specified side, using *side* values from **w**, **e**,
+        **s**, or **n**. The option is repeatable to set aside space on more
+        than one side. Such space will be left untouched by the main map
+        plotting but can be accessed by modules that plot scales, bars, text,
+        etc.  This setting overrides the common clearances set by **clearance**
+        in the initial :meth:`pygmt.Figure.subplot` call.
+
+    {V}
+    """
+    kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
+    kwargs["A"] = f'"{kwargs.get("A")}"' if kwargs.get("A") else None
+    # convert tuple or list to comma-separated str
+    panel = ",".join(map(str, panel)) if is_nonstr_iter(panel) else panel
+
+    with Session() as lib:
+        arg_str = " ".join(["set", f"{panel}", build_arg_string(kwargs)]).strip()
```

Comment:
Is `.strip()` necessary?
```suggestion
        arg_str = " ".join(["set", f"{panel}", build_arg_string(kwargs)])
```